lib/src/user_agent_application.dart

```diff
@@ -71,16 +69,13 @@ class UserAgentApplication {
   Future<AuthResponse> acquireTokenPopup(AuthRequest request) async {
     final response = await _convertMsalPromise<interop.AuthResponse>(
         _callJsMethod(() => _jsObject.acquireTokenPopup(request._jsObject)));
-
```

Comment:
Extremely small nitpick, but I don't understand the removal of the whitespace here and a few other places in this file.

Reply:
it's something intellij's autoformatting understands, not me :)
will remove

Reply:
conversely, I don't quite understand it's presence there. whitespaces inside the method are necessary only for separating parts of it to simplify understanding (which is a problem by itself), but these methods are usually 3 lines long or so, which is something a reader can understand easily

will do as you say ofc, as style is a prerogative of the author, but understanding why won't hurt - that's why I'm asking

Reply:
I suppose it is a personal preference. I find code a little harder to read when there isn't an empty line after a statement that wraps to multiple lines, like here. You are right though, these methods are very small and readable either way. I'm honestly curious what others would prefer here, I haven't given it much thought.

Regardless, I'd prefer that PRs don't contain small cosmetic changes to code that isn't changed otherwise.